bookmarks/services/tasks.py

```diff
@@ -161,6 +161,40 @@ def schedule_refresh_favicons(user: User):
         _schedule_refresh_favicons_task(user.id)
 
 
+def schedule_refresh_descriptions(user: User):
+    _schedule_refresh_descriptions_task(user.id)
+
+
+@background()
+def _schedule_refresh_descriptions_task(user_id: int):
+    user = get_user_model().objects.get(id=user_id)
+    bookmarks = Bookmark.objects.filter(owner=user)
+    tasks = []
+
+    for bookmark in bookmarks:
+        task = Task.objects.new_task(task_name='bookmarks.services.tasks._load_description_task', args=(bookmark.id,))
+        tasks.append(task)
+
+    Task.objects.bulk_create(tasks)
+
+
+@background()
+def _load_description_task(bookmark_id: int):
+    try:
+        bookmark = Bookmark.objects.get(id=bookmark_id)
+    except Bookmark.DoesNotExist:
+        return
+
+    logger.info(f'Load description for bookmark. url={bookmark.url}')
+
+    description = load_website_metadata(bookmark.url).description
+
+    if description is not None and description != bookmark.description:
+        bookmark.description = metadata.description
```

Comment:
🌱  `metadata` is not defined here. You can just do `bookmark.description = description` (same for the log afterwards)